src/Hl7.Fhir.Support.Poco/Specification/Source/CommonFhirPackageSource.cs

```diff
@@ -276,21 +276,20 @@ public IEnumerable<string> ListResourceUris(string? filter = null)
         }
 
         ///<inheritdoc/>
-        public async Task<IEnumerable<Resource>?> FindConceptMaps(string? sourceUri = null, string? targetUri = null)
+        public async Task<IEnumerable<Resource>> FindConceptMaps(string? sourceUri = null, string? targetUri = null)
```

Comment:
This is a breaking change, right?

Reply:
When yes, then we should add this change in https://github.com/FirelyTeam/firely-net-sdk/wiki/Breaking-changes-in-5.0

Reply:
I think so - but it's like the trick we use at HL7 now and then: the intention of these interface methods was to return an empty collection (the original implementers of this interface do so) - so this was a bug ;-)

When I thought about the consequences I figured it might mean there are now some null checks that will never be hit, but all the logic working on lists will probably still function perfectly fine.

I'll mark this PR as breaking, and update the wiki!